blog/2024-12-15-openhab-4-3-release.md

```diff
@@ -0,0 +1,294 @@
+This is especially useful for actions that have a significant impact, like opening your garage door.
+
+Action confirmation can be enabled by setting the `actionConfirmation` parameter.
+The dialog can be customized by setting the `type`, `title`, and `text` parameters:
+
+<br>
+<br>
+
+<div align="center" class="row">
+  <img style="max-width: 30%;" src="/uploads/2024-12-15-openhab-4-3-release/action-confirmation-basic.png"/>
+  <img style="max-width: 30%;" src="/uploads/2024-12-15-openhab-4-3-release/action-confirmation-dialog.png"/>
+  <img style="max-width: 30%;" src="/uploads/2024-12-15-openhab-4-3-release/action-confirmation-sheet.png"/>
+</div>
+
+<details>
+
+<summary>Show me the widget code for these examples</summary>
+
+Have a look at the following example to learn about the different options:
+
+```yaml
+uid: actionConfirmation
+tags: []
+props:
+  parameters: []
+  parameterGroups: []
+timestamp: Dec 10, 2024, 10:15:02 PM
+component: oh-card
+config:
+  title: Action Confirmation
+slots:
+  content:
+    - component: oh-button
+      config:
+        text: Confirm
+        action: url
+        actionUrl: https://www.openhab.org/blog/2024-12-15-openhab-4-3-release.html
+        actionConfirmation: Are you sure you want to open the openHAB blog?
+    - component: oh-button
+      config:
+        text: Confirm with Dialog
+        action: url
+        actionUrl: https://www.openhab.org/blog/2024-12-15-openhab-4-3-release.html
+        actionConfirmation:
+          type: dialog
+          title: openHAB Blog
+          text: Are you sure you want to open the openHAB blog?
+    - component: oh-button
+      config:
+        text: Confirm with Sheet
+        action: url
+        actionUrl: https://www.openhab.org/blog/2024-12-15-openhab-4-3-release.html
+        actionConfirmation:
+          type: sheet
+          text: Open the openHAB blog
+          color: green
+```
+
+Note that this example already uses the [new `oh-card` component](#introducing-oh-card-a-shared-base-for-card-widgets).
+
+</details>
+
+## Blockly Enhancements
+
+_Stefan Höhn ([@stefan-hoehn](https://github.com/stefan-hoehn)), Blockly Maintainer_
+
+## Sitemap Enhancements
+
+_Laurent Garnier ([@lolodomo](https://github.com/lolodomo)), openHAB Maintainer_
+
+## Basic UI Enhancements
+
+_Laurent Garnier ([@lolodomo](https://github.com/lolodomo)), openHAB Maintainer_
+
+## Add-on Enhancements
+
+openHAB 4.3 brings XX new add-ons, adding support for new devices:
+
+But not only were new bindings added, some existing bindings also received massive improvements:
+
+- The [JavaScript Scripting automation](/addons/automation/jsscripting) has finally fixed the remaining reported multi-threading issues for UI-bases rules.
+- The [KNX binding](/addons/bindings/knx) provides better support for KNX Secure, allowing to include keyring files and reading KNX Data Secure packets.
```

Comment:
```suggestion
- The [KNX binding](/addons/bindings/knx) provides better support for KNX Secure, allowing to include keyring files and decoding KNX Data Secure packets.
```
_Decoding_ might be a better choice compared to _reading_ to avoid confusion. Actually, actively reading datapoints (as well as writing) is what is still missing. We can just _listen_ to the data on the bus.